packages/frontend/src/components/common/ResourceView/resourceUtils.ts

```diff
@@ -54,6 +55,9 @@ export const getResourceUrl = (projectUuid: string, item: ResourceViewItem) => {
         case ResourceViewItemType.DASHBOARD:
             return `/projects/${projectUuid}/dashboards/${item.data.uuid}/view`;
         case ResourceViewItemType.CHART:
+            if (item.data.source === ChartSourceType.SQL) {
+                return `/projects/${projectUuid}/sql-runner-new/saved/${item.data.uuid}`;
```

Comment:
perhaps `item.data.slug` instead?

Reply:
it seems slug is not currently exposed in `content` :eyes:

Reply:
I see 👁️
Currently, links only work with `slug`, so having the `uuid` won't work - **yet**

Perhaps this should be a ticket